tests/run-make/ferrocene/compiler-arguments/upper_c_debuginfo/upper_c_debuginfo_multiple/Makefile

```diff
@@ -1,3 +1,6 @@
+# ignore-apple
+#   On Mac, signatures mean that the diff'd files are not equal when we expect them to be.
```

Comment:
This should be in the same line, as otherwise it's not marked as the reason for the ignore (applies to all similar changes).